src/components/backend-ai-edu-applauncher.ts

```diff
@@ -157,7 +157,6 @@ export default class BackendAiEduApplauncher extends BackendAIPage {
   async _initClient(apiEndpoint: string) {
     this.notification = globalThis.lablupNotification;
     const webUIShell: any = document.querySelector('#webui-shell');
-    // webUIShell.appBody.style.visibility = 'visible';
```

Comment:
Why did you annotate this line without erasing it?

Reply:
It looks like the code was cleared, not annotated out.